rows/__init__.py

```diff
@@ -29,6 +29,7 @@
 from rows.plugins._json import import_from_json, export_to_json
 from rows.plugins.csv import import_from_csv, export_to_csv
 from rows.plugins.txt import export_to_txt
+from rows.plugins._pandas import import_from_pandas
```

Comment:
As `pandas` is an optional dependency to `rows`, you need to put this import inside a `try`/`except` clause. Need also to import `export_to_pandas`.